pkg/app/server.go

```diff
@@ -371,3 +395,12 @@ func buildMetricsServer(m *metricshandler.MetricsHandler, durationObserver prome
 	})
 	return mux
 }
+
+func md5HashAsMetricValue(data []byte) float64 {
+	sum := md5.Sum(data) //nolint:gosec
+	// We only want 48 bits as a float64 only has a 53 bit mantissa.
```

Comment:
Would _significand_ be a better term to use here (in place of mantissa)? PTAL https://developer.apple.com/documentation/swift/float/significand.

Reply:
I prefer mantissa.

Reply:
I took the function from the alertmanager repository , so I would suggest to keep the delta to a minimum